commands/ci/status/status.go

```diff
@@ -82,6 +82,10 @@ func NewCmdStatus(f *cmdutils.Factory) *cobra.Command {
 						var status string
 						switch s := job.Status; s {
 						case "failed":
+							if job.AllowFailure {
+								status = utils.Yellow(s)
+								break
+							}
 							status = utils.Red(s)
```

Comment:
Having a conditional break inside a switch which is inside a nested loop looks quite ambiguous IMO.

I would go with this
```suggestion
							if job.AllowFailure {
								status = utils.Yellow(s)
							} else {
								status = utils.Red(s)
							}
```

Reply:
Yeah, that is more readable. Changed this